### ISSUE TITLE
Displaying emails

### ISSUE THREAD

**Issue by sakinshrestha** [puikinsh/shapely#121]
* When displaying emails, make sure you pass them via `antispambot()` function so that bots cannot recognize them.
* E.g: line 314-315 in `extras.php` becomes
     `<a class="author-email"
                       href="mailto:<?php echo esc_attr( antispambot( $author_email ) ); ?>"><?php echo esc_html( $author_email ); ?></a>`
* Same issue in line 402

**Comment by justintadlock** [comment]
I'd wrap it in both the attribute and link text like so:

```
<a class="author-email" href="mailto:<?php echo esc_attr( antispambot( $author_email ) ); ?>"><?php echo esc_html( antispambot( $author_email ) ); ?></a>
```

**Comment by cristianraiber** [comment]
Fixed in: 188b1d659a6a3f777e91e37b3e20572394338b70
